### DISCUSSION TITLE
How do I get the console to recognize my anthropic pro plan?

### DISCUSSION THREAD

**Discussion by Rick-Kennerly** [logancyang/obsidian-copilot#754] [General]
I'm trying to use the Co-Pilot API in Obsidian via Anthropic.  I've created an Anthropic Pro Plan and have a reciept, so I know the charge went through.  I created an API key through my pro plan account.  I paste that key into the proper slot in Co Pilot to index my vault.  However, I get a message saying I don't have enough credits.  When I click the link back to the Anthropic Console, I see my API key, but there is no evidence of my Paid Pro Plan.  All I get are screens wanting me to buy credits by entering a credit card.  How do I get the console to recognize my pro plan?

**Comment by logancyang** [comment]
You have to get credit to use Anthropic API nowadays it seems, the Pro plan is just web chat, same as ChatGPT plus.

This is an FAQ. Pro and API are 2 different things, you don't need to buy pro to use API, you need credit and pay as you go. That's the beauty of API.
